rele/config.py

```diff
@@ -4,7 +4,12 @@
 
 from google.oauth2 import service_account
 
-from .client import DEFAULT_ACK_DEADLINE, DEFAULT_ENCODER_PATH, get_google_defaults
+from .client import (
+    DEFAULT_ACK_DEADLINE,
+    DEFAULT_BLOCKING,
+    DEFAULT_ENCODER_PATH,
+    get_google_defaults,
```

Comment:
`isort` is complaining about the trailing comma in the last import statement. Please remove it in order for the CI pipeline to pass

```suggestion
    get_google_defaults
```